Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -677,6 +677,13 @@ public void disableSmartSession() {
     }
 
     /**
+     * <p>Add key value pairs to all requests</p>
+     */
+    public void addMetadataToRequests(@NonNull String key, Object value) {
```

Comment:
there's `addMetadataToRequests` here, then in Pref helper there's `setRequestMetadata`. is there a reason why these methods have slightly different signatures?